toolchain/fortify-headers/Makefile

```diff
@@ -8,12 +8,13 @@ include $(TOPDIR)/rules.mk
 include $(INCLUDE_DIR)/target.mk
 
 PKG_NAME:=fortify-headers
-PKG_VERSION:=1.1
```

Comment:
You can leave PKG_VERSION if it is indeed 1.1, PKG_SOURCE_VERSION just points to its hash